### PR TITLE
Refactoring local builder image to run as local user

### PR DESCRIPTION
**Pull request by etsauer**
#### What is this PR About?
Fixes the test image

#### How should we test or review this PR?
```
cd  ./openshift-playbooks/
docker build -t jekyll-local-builder -f container-images/local-builder/Dockerfile .
docker run -u `id -u` -v $PWD/:/home/jekyll/src/:Z -p 4000:4000 -it jekyll-local-builder
```

#### Is there a relevant Trello card or Github issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

### CODE REVIEW

**Comment by etsauer** [comment]
@beelandc made some more significant changes. Mind re-reviewing?

File: README.adoc [hunk 1]
```diff
@@ -19,7 +19,11 @@ git clone https://github.com/redhat-cop/openshift-playbooks.git
 +
 ----
 cd openshift-playbooks
-docker-compose up -d
+docker run \
```
Comment (beelandc) [inline README.adoc]: Can you add a quickstart step around the docker build the builder image?

**Comment by sabre1041** [comment]
@etsauer Build was successful

Docker info below
```
Client:
 Version:      1.13.1
 API version:  1.26
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 08:47:51 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      1.13.1
 API version:  1.26 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   092cba3
 Built:        Wed Feb  8 08:47:51 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

File: README.adoc (hunk 1 above)
Comment (etsauer) [reply README.adoc]: @beelandc once merged, we will build the image and publish it in docker hub, so user's won't need to run a docker build.

**Comment by etsauer** [comment]
@beelandc see my comment. once we have your approval we will merge this.

File: README.adoc (hunk 1 above)
Comment (beelandc) [reply README.adoc]: ah, gotcha

**Pull request merged**